vue-frontend/src/components/partials/ScreenHeaderV2.vue

```diff
@@ -100,38 +100,45 @@ export default {
           className:
             " tw-mt-0 tw-mb-0 tw-ml-0 tw-p-0 tw-text-black-900/80 hover:tw-text-black-900/80 tw-mr-[20px] sm:tw-mr-[40px] tw-inline-block tw-relative after:tw-absolute after:tw-w-2/4 after:tw-scale-x-0 after:tw-h-0.5 after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-black-900 after:tw-origin-bottom-left after:tw-duration-300 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left",
           target: false,
+          event: "tap_jobs_header_home",
```

Comment:
You have to check `isJobsUrl` here.
But have to add condition in `sendEvent` that `event != " "`